lib/my_obfuscate.rb

```diff
@@ -80,12 +97,15 @@ def missing_column_list(table_name, columns)
   end
 
   def check_for_table_columns_not_in_definition(table_name, columns)
-    missing_columns = missing_column_list(table_name, columns)
-    unless missing_columns.length == 0
-      error_message = missing_columns.map do |missing_column|
-        "Column '#{missing_column}' defined in table '#{table_name}', but not found in table definition, please fix your obfuscator config."
-      end.join("\n")
-      raise RuntimeError.new(error_message)
+    unless unspecified_columns_behavior == :ignore
+      missing_columns = missing_column_list(table_name, columns)
+      unless missing_columns.length == 0
```

Comment:
Might be more readable as `unless missing_columns.length == 0 && unspecified_columns_behavior != :ignore`